src/runtime/virtcontainers/qemu.go

```diff
@@ -1886,6 +1894,14 @@ func (q *qemu) hotplugNetDevice(ctx context.Context, endpoint Endpoint, op Opera
 			}
 		}()
 
+		// Hotplug net dev to pcie root port for QemuVirt
```

Comment:
Ah, is this why we need  `defaultVirtPcieRootPortNum = 1` is the net device configurable, what if we have several net devices?